pom.xml

```diff
@@ -191,6 +191,12 @@
       <version>1.74</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-codec-http</artifactId>
+      <version>4.1.107.Final</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
It is not up to us. They even have dependency checkstyle addition to coding style where if you use import which is not declared. it should be declared and define exact scope of dependency,
[ERROR] Used undeclared dependencies found: [ERROR]    io.netty:netty-codec-http:jar:4.1.107.Final:compile [ERROR] Non-test scoped test only dependencies found: [ERROR]    io.netty:netty-codec-http:jar:4.1.107.Final:compile

From my prespective having coding style is not good approach overall because of forcing to change one file many times, leading to extra effort. 

I can two things 
1) delete dependency check style
2) Keep as it was
3) exclude this library from io vertx 
third option is not wise to do.

Reply:
@julianladisch, what do you suggest we do regarding this dependency?